multimodal/src/autogluon/multimodal/utils/data.py

```diff
@@ -515,3 +526,38 @@ def infer_dtypes_by_model_names(model_config: DictConfig):
         fallback_dtype = list(allowable_dtypes)[0]
 
     return allowable_dtypes, fallback_dtype
+
+
+def contains_valid_images(data: Union[str, list], sample_n: Optional[int] = 50) -> bool:
```

Comment:
How about reusing `is_imagepath_column()` in `infer_types.py`?

Reply:
the is_imagepath_column checks on a df.Series. This test was for array of images. That being said, let me check if the `image` header is needed. If not, I can convert the image array to pd.Series and re-use `is_imagepath_column()`.